packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1068,6 +1112,9 @@ export class DeltaManager
             } else if (connection.mode !== "write") {
                 this.fetchMissingDeltas("AfterConnection", this.lastQueuedSequenceNumber);
             }
+        } else {
+            this.connectionStateProps.connectionInitialOpsFrom = initialMessages[0].sequenceNumber;
+            this.connectionStateProps.connectionInitialOpsTo = last + 1;
```

Comment:
Why the `+ 1` here? Feels like it could be misleading since `initialMessages` wouldn't have included that seq num

Reply:
All the code recently switched to reporting [left, right), i.e. left is inclusive, right is exclusive.
It used to be both exclusive (which I really disliked :)).
We can change it to be inclusive on both sides, though that is engraved into APIs across layers, so would be a bit hard to change (so likely it's better to change only in telemetry).